modules/data-loading/pages/data-streaming-connector.adoc

```diff
@@ -400,12 +332,293 @@ GSQL > RUN LOADING JOB load_person
+
+
+
+|`<prefix>.sasl.jaas.config`
+|JAAS login context parameters for SASL connections in the format used by JAAS configuration files.
+See link:https://docs.oracle.com/javase/8/docs/technotes/guides/security/jgss/tutorials/LoginConfigFile.html[JAAS Login Configuration File] for
+
+
+|`<prefix>.ssl.endpoint.identification.algorithm`
+|
+The endpoint identification algorithm to validate server hostname using server certificate. Default is https. Disable server host name verification by setting `ssl.endpoint.identification.algorithm` to an empty string
+
+
+|`<prefix>.ssl.keystore.location`
+|The location of the key store file.
+
+
+|`<prefix>.ssl.keystore.password`
+|The password of the key store file.
+
+
+|`<prefix>.ssl.key.password`
+|The password of the private key in the key store file or the PEM key specified in `ssl.keystore.key`.
+
+
+|`<prefix>.ssl.truststore.location`
+|The location of the trust store file.
+
+|`<prefix>.ssl.truststore.password`
+|The password for the trust store file.
+
+|===
+
+==== Subsection configurations
+The subsection configurations come after the other configurations, and specify the names of the connectors you want to create using the provided configurations before the subsections, as well as the maximum number of tasks that can be created for the connector.
+
+You can create one or more connectors from one configuration file by providing a subsection for each connector you want to create.
+In the case of the MirrorMaker connector class used in this procedure, *we suggest you only create one connector from each file*, due to limited available configurations in the subsection.
+If you create multiple connectors, they will have almost identical properties.
+
+Start a subsection with a section title enclosed in square brackets.
+This title has no effect on the connector to be created, but the titles need to be unique in the configuration file.
+
+The following subsection configurations are available:
+
+
+|===
+|Field |Description
+
+|`name`
+|Name of the connector.
+
+|`tasks.max`
+|
+The maximum number of tasks that to be created for this connector. The connector may create fewer tasks if it cannot achieve this level of parallelism.
+The value for this configuration should not be larger than the number of partitions of the topic(s) in source Kafka.
+The default value is 1.
+|===
+
+
+==== Example
+The following is an example configuration file:
+
+[,text]
+----
+include::example$config-avro[]
+----
+
+The file in the example creates a connector named `avro-test-without-registry` with all configurations specified in the file.
+
+include::partial$create-connector.adoc[]
+
+include::partial$create-data-source.adoc[]
+
+include::partial$create-loading-job-kafka.adoc[]
+
+For example, suppose we have the following graph schema with only one vertex type
+ and the following JSON line data in Kafka:
```

Comment:
It would be better to make it explicit that the data is in tigergraph's Kafka instead of external one.